src/elements/Header/_Header.js

```diff
@@ -1,17 +1,22 @@
-import _ from 'lodash'
-import React, { Component, PropTypes } from 'react'
-import classNames from 'classnames'
+import cx from 'classnames'
```

Comment:
Definitely, I've been skipping all optimizations while we crank toward 1.0.  This is a great point.